exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileWrapper.java

```diff
@@ -60,7 +60,7 @@
   private final HashMap<String, Long> majorFragmentTallyMap;
   private final long majorFragmentTallyTotal;
   private final OptionList options;
-  private final boolean onlyImpersonationEnabled;
+  private final boolean isOnlyImpersonationEnabled;
```

Comment:
usually the convention is that boolean variable name should not start with `"is"`, it's the getter function which should have that. Variable name should only be `onlyImpersonationEnabled`.

Reply:
Done.